ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -23,9 +23,90 @@ const wreqr = require('../../exports/wreqr.js')
 const Router = require('../router/router.js')
 const announcement = require('../announcement')
 const Common = require('../../js/Common.js')
+import styled from '../../react-component/styles/styled-components'
+
+const FormTitle = styled.h3`
+  padding-bottom: ${props => props.theme.minimumSpacing};
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: inline-block;
+  width: 100%;
+`
+
+const FormContents = styled.span`
+  display: block;
+`
+
+const NewFormCircle = styled.div`
+  font-size: calc(3 * ${props => props.theme.largeFontSize});
+  padding-top: ${props => props.theme.minimumSpacing};
+`
+
+const DefaultIcon = styled.div`
+  position: absolute;
+  right: 0;
+  top: 0;
+`
+
+const CustomSearchForm = props => {
+  const { title, createdOn, createdBy, isDefault } = props
+  return (
+    <div style={{ position: 'relative', height: '100%' }}>
```

Comment:
We're trying to stick to using styled components instead of inlining styles.  

I'm a little curious about the height being 100%, is the height being set to a specific value on the parent component that this is growing to?